src/modules/missionHelper/i18n/sv_SE.json

```diff
@@ -1,8 +1,8 @@
 {
-	"diyMission": "Det här uppdraget verkar vara ett storskaligt alliansuppdrag!",
-	"expansions": "Uppdrag att utöka:",
-	"followups": "Uppföljningsuppdrag | Uppföljningsuppdrag",
-	"generated_by": "Skapad av",
+	"diyMission": "This mission seems to be a large scale alliance mission!",
+	"expansions": "Spread | Spreads",
+	"followups": "Follow-up Mission | Follow-up Missions",
+	"generated_by": "Generated by",
```

Comment:
this seems wrong (sv_SE → en_XX)?
Follow-Up Missions are called `Påöljande uppdrag vid avklarat uppdrag` in the official mission help of the game.